utility.lua

```diff
@@ -2984,3 +2984,30 @@ function Auxiliary.ThisCardInGraveAlreadyReset2(e)
 	e1:Reset()
 	e:Reset()
 end
+--Player p place n cards from his hand on the bottom of his Deck in any order
+function Auxiliary.PlaceOnBottom(p,n)
+	local g=Duel.GetMatchingGroup(Card.IsAbleToDeck,p,LOCATION_HAND,0,nil)
+	if g:GetCount()<n then return end
+	Duel.ShuffleHand(p)
+	Duel.BreakEffect()
+	Duel.Hint(HINT_SELECTMSG,p,HINTMSG_TODECK)
+	local sg=g:Select(p,n,n,nil)
```

Comment:
We should support _Dark Spirit's Mastery_ by allow passing `sg` as param

Reply:
or another function to simply Tearlaments scripts

Reply:
So the function should be something like:
```lua
function Auxiliary.PlaceGroupOnDeckBottom(p,sg)
end
```

The script should generate sg by themselves.
Is it right?

Reply:
I think so